pkg/constants/constants.go

```diff
@@ -87,6 +87,9 @@ const (
 	// OSMControllerName is the name of the OSM Controller (formerly ADS service).
 	OSMControllerName = "osm-controller"
 
+	// OSMInjectorName is the name of the OSM Injector.
+	OSMInjectorName = "osm-injector"
```

Comment:
Where is this being used? I don't see its usage in this PR.

Reply:
Yeah agree. I'll use that instead.